sklearn/ensemble/bagging.py

```diff
@@ -304,6 +305,9 @@ def _fit(self, X, y, max_samples, max_depth=None, sample_weight=None):
         if not (0 < max_samples <= X.shape[0]):
             raise ValueError("max_samples must be in (0, n_samples]")
 
+        self._max_samples = max_samples
```

Comment:
I don't understand how this test would work as `_max_samples` doesn't even exist unless `_fit()` is called.